wgpu/Cargo.toml

```
@@ -283,8 +283,7 @@ web-sys = { version = "0.3.53", features = [
 ]}
 js-sys = "0.3.50"
 wasm-bindgen-futures = "0.4.23"
-# enable parking_lot's wasm-bindgen feature so that it, in turn, enables that of crate `instant`
-parking_lot = { version = "0.11", features = ["wasm-bindgen"] }
+parking_lot = { version = "0.12" }
```

Could we just move this back to 

```suggestion
parking_lot = "0.12"
```

---

Indeed, done!